src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -1103,13 +1103,13 @@ public static String escapeXpathValue(String value) {
 		return String.format("'%s'", value);
 	}
 
-	private boolean textContains(String actual, String text, boolean ignoreCase) {
+	public static boolean textContains(String actual, String text, boolean ignoreCase) {
 		return ignoreCase
 				? StringUtils.containsIgnoreCase(actual, text)
 				: StringUtils.contains(actual, text);
 	}
 
-	private boolean textIs(String actual, String text, boolean ignoreCase) {
+    public static boolean textIs(String actual, String text, boolean ignoreCase) {
```

Comment:
It seems like indentation here is incorrect.

Reply:
I fixed te indentation. For some reason, this one was using tabs instead of spaces. the other indentations all worked with tabs...